### PR TITLE
Feature/forgot password link

### PR DESCRIPTION
**Pull request by J01716**
### ⁉️ Related Issue

Closes: #736

## 📖 Description

Added text container at the bottom right of the password input in the login view.
Added method that opens URL received as parameter and call that method from the added text container with the link to the signets password change service : https://signets-ens.etsmtl.ca/Public/MotDePassePerdu.aspx

### 🧪 How Has This Been Tested?

The change was tested manually.

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [ ] If needed, I added analytics.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`. 
- [ ] Make sure golden files changes were reviewed and approved.

### 🖼️ Screenshots (if useful):

image
image
image

### CODE REVIEW

**Review by eneiss — DISMISSED** [review]
LGTM, good job 🚀

**Review by camillebrulotte — COMMENTED** [review]
LGTM mais il semble avoir 3 tests qui ne passent plus sinon good job! :D https://github.com/ApplETS/Notre-Dame/actions/runs/4085632311/jobs/7043861447

**Review by eneiss — DISMISSED** [review]
LGTM! Good job 🔥

**Review by camillebrulotte — DISMISSED** [review]
Après le petit conflit, LGTM ! :D

**Review by AntoineLemelin — DISMISSED** [review]
LGTM

**Review by MysticFragilist — APPROVED** [review]
LGTM! :fire:

**Review by AntoineLemelin — APPROVED** [review]
LGTM!

File: lib/ui/views/login_view.dart [hunk 1]
```diff
@@ -252,4 +273,41 @@ class _LoginViewState extends State<LoginView> {
       locator<AnalyticsService>().logError("login_view", "Cannot send email.");
     }
   }
+
+  //From web_link_card_viewmodel
+  Future<void> launchInBrowser(String url) async {
```
Comment (XavierPaquet-Rapold) [inline lib/ui/views/login_view.dart:278]: Étant donné que c'est dupliqué d'un autre fichier, que penses-tu de créer un service `BrowserService` ?

**Review by XavierPaquet-Rapold — CHANGES_REQUESTED** [review]

File: lib/core/services/launch_url_in_browser_service.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// FLUTTER / DART / THIRD-PARTIES
```
Comment (XavierPaquet-Rapold) [inline lib/core/services/launch_url_in_browser_service.dart:1]: Penses-tu qu'on pourrait l'ajouter dans la classe `LaunchUrlService` ?

**Review by XavierPaquet-Rapold — CHANGES_REQUESTED** [review]

**Review by eneiss — APPROVED** [review]
I noticed a hard-coded light theme in the safari viewer that could be improved, otherwise LGTM 💯

**Comment by camillebrulotte** [comment]
Je vais reprendre cette PR

File: lib/core/services/launch_url_service.dart [hunk 3]
```diff
@@ -1,14 +1,55 @@
...
+          'com.sec.android.app.sbrowser',
+          // ref. https://play.google.com/store/apps/details?id=com.microsoft.emmx
+          'com.microsoft.emmx',
+          // https://play.google.com/store/apps/details?id=com.UCMobile.intl
+          'com.UCMobile.intl',
+        ],
+      ),
+      safariVCOption: const custom_tabs.SafariViewControllerOption(
+        preferredBarTintColor: AppTheme.etsLightRed,
+        preferredControlTintColor: AppTheme.lightThemeBackground,
```
[the remark this answers is not in the document]
Comment (camillebrulotte) [reply lib/core/services/launch_url_service.dart:47]: Je vais regarder ça !

**Review by Udito3 — DISMISSED** [review]
LGTM! :)

**Review by JonathanDuvalV — APPROVED** [review]
LGTM

File: lib/core/constants/urls.dart [hunk 4]
```diff
@@ -9,4 +9,6 @@ class Urls {
   static const String clubYoutube =
       "https://youtube.com/channel/UCiSzzfW1bVbE_0KcEZO52ew";
   static const String clubDiscord = "https://discord.gg/adMkWptn6Y";
+  static const String signetsForgottenPword =
```
Comment (MysticFragilist) [inline lib/core/constants/urls.dart:12]:
```suggestion
  static const String signetsForgottenPassword =
```

**Review by MysticFragilist — CHANGES_REQUESTED** [review]

**Review by JonathanDuvalV — APPROVED** [review]

**Review by MysticFragilist — APPROVED** [review]
LGTM! :smile:

**Review by Udito3 — APPROVED** [review]
LGTM!

**Pull request merged**